

This model is from the article:  
**Genetic and metabolic determinants of nutritional phenotype in an insect-bacterial symbiosis.**   
MacDonald SJ, Thomas GH, Douglas AE. _Mol Ecol._ 2011 May;20(10):2073-84.
[21392141](http://www.ncbi.nlm.nih.gov/pubmed/21392141) ,  
**Abstract:**   
The pervasive influence of resident microorganisms on the phenotype of their
hosts is exemplified by the intracellular bacterium Buchnera aphidicola, which
provides its aphid partner with essential amino acids (EAAs). We investigated
variation in the dietary requirement for EAAs among four pea aphid
(Acyrthosiphon pisum) clones. Buchnera-derived nitrogen contributed to the
synthesis of all EAAs for which aphid clones required a dietary supply, and to
none of the EAAs for which all four clones had no dietary requirement,
suggesting that low total dietary nitrogen may select for reduced synthesis of
certain EAAs in some aphid clones. The sequenced Buchnera genomes showed that
the EAA nutritional phenotype (i.e. the profile of dietary EAAs required by
the aphid) cannot be attributed to sequence variation of Buchnera genes coding
EAA biosynthetic enzymes. Metabolic modelling by flux balance analysis
demonstrated that EAA output from Buchnera can be determined precisely by the
flux of host metabolic precursors to Buchnera. Specifically, the four EAA
nutritional phenotypes could be reproduced by metabolic models with unique
profiles of host inputs, dominated by variation in supply of aspartate,
homocysteine and glutamate. This suggests that the nutritional phenotype of
the symbiosis is determined principally by host metabolism and transporter
genes that regulate nutrient supply to Buchnera. Intraspecific variation in
the nutritional phenotype of symbioses is expected to mediate partitioning of
plant resources among aphid genotypes, potentially promoting the genetic
subdivision of aphid populations. In this way, microbial symbioses may play an
important role in the evolutionary diversification of phytophagous insects.

This model originates from BioModels Database: A Database of Annotated
Published Models (http://www.ebi.ac.uk/biomodels/). It is copyright (c)
2005-2011 The BioModels.net Team.  
To the extent possible under law, all copyright and related or neighbouring
rights to this encoded model have been dedicated to the public domain
worldwide. Please refer to [CC0 Public Domain
Dedication](http://creativecommons.org/publicdomain/zero/1.0/) for more
information.

In summary, you are entitled to use this encoded model in absolutely any
manner you deem suitable, verbatim, or with modification, alone or embedded it
in a larger context, redistribute it, commercially or not, in a restricted way
or not..  
  
To cite BioModels Database, please use: [Li C, Donizelli M, Rodriguez N,
Dharuri H, Endler L, Chelliah V, Li L, He E, Henry A, Stefan MI, Snoep JL,
Hucka M, Le Nov√®re N, Laibe C (2010) BioModels Database: An enhanced, curated
and annotated resource for published quantitative kinetic models. BMC Syst
Biol., 4:92.](http://www.ncbi.nlm.nih.gov/pubmed/20587024)

